src/style.css

```
@@ -0,0 +1,123 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.wrap {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  /* flex-wrap: wrap; */
```

just add wrap, than you wouldn't need column direction in lower breakpoints
```suggestion
  flex-wrap: wrap;
```